nano/templates/gps.tmpl

```diff
@@ -24,11 +24,15 @@ Used In File(s): \code\modules\telesci\gps.dm
 </div>
 
 <h3>Detected signals</h3>
+<div class="item">
+	<div class="itemLabelNarrow">{{:data.gpstag}}</div>
+	<div class="itemContentWide">{{:data.location_text}}</div>
+</div>
 {{for data.devices}}
 	<div class="item">
 		<div class="itemLabelNarrow">{{:value.tag}}</div>
```

Comment:
If nano has `continue` you could make it skip your own gps on the main list, to reduce bloat.

Reply:
This is done already in DM